src/private/zerocopy.rs

```diff
@@ -83,7 +83,7 @@ where
     }
 }
 
-unsafe impl<T> Send for Receiver<T> where T: Send + 'static {}
+unsafe impl<'a, T> Send for Receiver<T> where T: Send + 'a {}
```

Comment:
Why is this necessary and what benefit does it bring? `T: 'static` is still throughout all of `Receiver` and `Channel` and I think removing it would require some very careful thinking, as that might be unsound.

Reply:
Got it - it is this one: https://github.com/rust-lang/rust/issues/64552